components/conversations/CommentReactionPicker.js

```diff
@@ -91,18 +115,59 @@ const getOptimisticResponse = (comment, emoji, isAdding) => {
   }
 };
 
+const getOptimisticResponseForUpdates = (update, emoji, isAdding) => {
```

Comment:
If we move everything to the same mutation, the diff between the two versions will be really small. Maybe we can have only one function like:

```suggestion
const getOptimisticResponseForUpdates = (entity, emoji, isAdding) => {
  const { __typename } = entity;
  const fieldName = __typename === 'Update' ? 'update' : 'comment';
```

Reply:
Done. 😄